src/content/styleguide.mdx

```diff
@@ -19,14 +19,15 @@ When naming files, you need to remember about few things.
 - Name the file the same as the main (default) export.
 
 The available file type suffixes (list is non-exhaustive):
-- `.component` for React components,
+- `.container` for connecting store to component (Redux's `connect`)
+- `.component` for standard React components,
 - `.styled` for Styled components,
 - `.stories` for files holding Storybook stories,
 - `.test` for unit tests,
 - `.e2e` for end-to-end tests,
 - `.hook` for hooks,
 - `.fragment` for GraphQL fragments,
-- `.interface` for interfaces,
+- `.types` for interfaces, types, enums, etc.
```

Comment:
So we agree on not distinguishing enums etc. in files with different extension like we are doing right now in mudita-center?

Reply:
TBH I'm kinda *meh* when it comes to using a sintle `.type` file. Having `.enum` and `.interface` adds more meaning to the structure.

Reply:
Yup, I think the same.

Reply:
Sorry, but I remember sth opposite and imho it's too late for such change. I was asking for some feedback few times and no one requested such change so I assume it is OK.

The most thing I don't like in this approach is that in fact, by the most time, we don't care if something is a type or interface or enum - it's just a typing of some objects / props etc. Very common situation is where we have only 1 interface, using only 1 enum and 1 type. So, should we create 3 separate files to properly type a single component? It's overkill.

Isn't one single file for all typings stuff enough? I would give it a try.

Reply:
Regarding it being too late, I don't think that's the case. You got feedback from Kamil as soon as he got back to work after the holiday's season, so it counts.

Reply:
Nvm, changed in b2fa83b